tests/jury/metrics/test_custom_bleu.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from jury import Jury
+from jury.metrics import AutoMetric
+from tests.jury import TEST_DATA_DIR
+from tests.jury.conftest import get_expected_output
+from tests.utils import assert_almost_equal_dict
+
+_CUSTOM_BLEU_PATH = TEST_DATA_DIR / "custom_bleu"
+
+
+@pytest.fixture(scope="module")
+def jury():
+    metric = AutoMetric.load(_CUSTOM_BLEU_PATH)
```

Comment:
I am confused with this. Previously, we were supplying the script path whereas now do we use the directory (or rather the package) path? If so, there are two modules under this package, one of which has the same name as the package. Is it like the datasets.load_dataset where we generally use the same name with the module and package to make loading easier?

Reply:
Yes, previously jury containing metric modules, rather than metric packages. Now, metrics are python packages for jury, and it is required to be the paths pointing to the package, and not modules. As you said, currently the behavior to load a metric is alike to datasets package's loading approach as in `load_dataset()` or `load_metric()`.